index.js

```diff
@@ -57,3 +63,17 @@ function tryParseFloat(value) {
 	}
 	return result;
 }
+
+function getTypeFromUnit ( unit ) {
+	if ( cssAngleUnits.indexOf(unit) !== -1 ) {
+		return 'angle';
+	} else if ( cssResolutionUnits.indexOf(unit) !== -1 ) {
```

Comment:
Yup, that is definitely more performant :) It’s just that I’m always looking for KISS approach, but this is definitely understandable when you closely look at it. I will apply your approach so we can have best possible solution.